update-vanadium-patches.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+gpwd=$(pwd)
+readonly path="./vanadium_patches/"
+readonly url=https://raw.githubusercontent.com/GrapheneOS/Vanadium/main/patches
+patches=() # current patch names
+smallpatches=() # truncated version of the patches for comparison
+vpatches=() # vanadium patches
```

Comment:
can you name these such that the comments aren't needed? even if the variable names are very long. improves readability.